### ISSUE TITLE
Create Space Element

### ISSUE THREAD

**Issue by ikeough** [hypar-io/sdk#49]
A Space is a volume created by extruding a set of co-planar polygons to a height, or by projecting a a set of co-planar polygons onto a plane. The first polygon provided in the set is understood to be the outer boundary of the space, with the ensuing polygons understood to be ordered in an CW direction representing "holes" in the space.

```
public Space(Polygon boundary, IEnumerable<Polygon> voids, double height){...}
public Space(Polygon boundary, IEnumerable<Polygon> voids, Plane plane){...}
```

**Comment by hummingburd** [comment]
Okay, I'll get on this today.

**Comment by ikeough** [comment]
Implemented in 07d6b63fee27d5699aebab5bc2d916101d1394ae.
